test/jtreg_test_thread_factory/src/share/classes/Virtual.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022, 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
```

Comment:
Hello Leonid, is this file expected to have a Classpath exception or should it have the same license header that we use for test files?